### PR TITLE
Added a CMakeLists.txt, and updated re2/googletest.

### PR DESCRIPTION
**Pull request by haberman**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+cmake_minimum_required (VERSION 2.6)
+project (Bloaty)
+
+set(RE2_BUILD_TESTING OFF CACHE BOOL "enable testing for RE2" FORCE)
+add_subdirectory(third_party/re2)
+include_directories(third_party/re2)
+
+include_directories(.)
```
Comment (xfxyjwf) [inline CMakeLists.txt]: include_directories(src)?

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (xfxyjwf) [inline CMakeLists.txt]: Remove this empty line?

Comment (haberman) [reply CMakeLists.txt]: Done.

File: CMakeLists.txt (hunk 1 above)
Comment (haberman) [reply CMakeLists.txt]: Doesn't seem to be required? It is compiling without this.

Comment (xfxyjwf) [reply CMakeLists.txt]: Hmm. I saw that in the source code it's doing 'include "bloaty.h"' so I assume you need to put src in the include path. Is it there by default?

**Pull request merged**